frontend/tailwind.config.ts

```diff
@@ -18,22 +18,22 @@ export default {
     maxHeight: standardSizes,
     maxWidth: standardSizes,
     borderRadius: {
-      sm: '4px',
-      md: '8px',
-      lg: '16px',
-      xl: '30px',
+      'sm': '4px',
+      'md': '8px',
+      'lg': '16px',
+      'xl': '30px',
       '2xl': '40px',
     },
     boxShadow: {
-      raised:
+      'raised':
         'rgba(0, 0, 0, 0.2) 0px 5px 5px -3px, rgba(0, 0, 0, 0.14) 0px 8px 10px 1px, rgba(0, 0, 0, 0.12) 0px 3px 14px 2px',
       'raised-tighter':
         'rgba(0, 0, 0, 0.1) 0px 3px 3px -2px, rgba(0, 0, 0, 0.08) 0px 6px 7px 0px, rgba(0, 0, 0, 0.06) 0px 2px 9px 1px',
-
     },
     colors: {
       'alert-color': 'rgb(var(--alert-color) / <alpha-value>)',
-      'alt-black': 'rgb(var(--alt-black) / <alpha-value>)',
+      // 'alt-black': 'rgb(var(--alt-black) / <alpha-value>)',
+      'alt-black': '#383838', // @BEN: ex.: how about this pattern to use tailwind theme config instead?
```

Comment:
i'd like to keep as is for now so that the design tokens (the actual color values) are defined outside of this config file. also, to allow tailwind opacity and other modifiers to work you have to use that strange rgb / alpha syntax

Reply:
ok, got it.